### ISSUE TITLE
[BUG] Writing Assertions snippet needs to be updated

### ISSUE THREAD

**Issue by Nav-2d** [microsoft/playwright#8333]
**Context:**
- Playwright Version: 1.14.0
- Operating System: Mac
- Node.js version: 12.20
- Browser: All

**Code Snippet**
N/A

**Describe the bug**
The `Writing Assertions` [snippet](https://playwright.dev/docs/intro#writing-assertions) needs to be updated

Expected:
The snippet test runs and fails on the last assertion since golden file for `get-started.png` snapshot is missing.

Actual:
Fails on first assertion. (Probably need to update title or use regex)
Fails on `page.locator` assertions since there are more than 1 locator matching the selector.

**Comment by JoelEinbinder** [comment]
The title is "Fast and reliable end-to-end testing for modern web apps | Playwright"

@pavelfeldman Do you want to change the title or the test?

**Comment by pavelfeldman** [comment]
I'd say change test to be regex /Playwright/?

**Comment by anishkny** [comment]
It looks like there are further failures due to locator strictness failure: 

```

    locator.getAttribute: Evaluation failed: Error: strict mode violation: selector resolved to 2 elements.
        at u.querySelector (<anonymous>:3:34807)
        at eval (eval at evaluate (:3:1339), <anonymous>:7:32)
        at i (<anonymous>:3:37685)
        at <anonymous>:3:37773
        at Object.run (<anonymous>:3:38253)
        at eval (eval at evaluate (:3:1339), <anonymous>:1:14)
        at t.default.evaluate (<anonymous>:3:1362)
        at t.default.<anonymous> (<anonymous>:1:44)
    =========================== logs ===========================
      retrieving attribute "href" from "text=Get Started"
      strict mode violation: selector resolved to 2 elements.
    ============================================================

       9 |
      10 |   // Expect an attribute "to be strictly equal" to the value.
    > 11 |   await expect(page.locator("text=Get Started")).toHaveAttribute(
         |                                                  ^
      12 |     "href",
      13 |     "/docs/intro"
      14 |   );
```

I have proposed a fix in #8372.

In the long run, perhaps doc examples should be automated tested? Any thoughts on how best to go about doing this?
